docs/v3/core/v3.0.rst

```diff
@@ -40,6 +40,8 @@ Status of this document
 
 ZEP0001 was accepted on May 15th, 2023 via https://github.com/zarr-developers/zarr-specs/issues/227.
 
+This specification is the latest version.
```

Comment:
```suggestion
```
If the spec says "this is the latest version", then we have to change the spec when we release a new version. We can prevent this by simply not stating "this is the latest version".

Reply:
I agree, but I think the author of the latest version should take responsibility for adding the text to avoid confusion, as seen in #332.